src/components/NowPlaying.tsx

```diff
@@ -178,38 +426,49 @@ class NowPlaying extends React.Component<{}, {audioFeatures: AudioFeaturesRespon
           {/* Statistic #1 */}
           <div className={styles.statContainer}>
             <div className={styles.statTextContainer}>
-              <div className={styles.text + styles.statLabel} style={{fontSize: "23px", color: "rgb(180,180,180)", fontWeight: "600"}}>{"Danceability"}</div>
-              <div className={styles.text + styles.statValue} style={{fontSize: "48px", color: "white", fontWeight: "500"}}>{"75"}</div>
+              <div className={styles.text + styles.statLabel} style={{fontSize: "23px", color: "rgb(200,200,200)", fontWeight: "600"}}>{"Danceability"}</div>
+              <div className={styles.text + styles.statValue} style={{fontSize: "48px", color: "white", fontWeight: "500"}}>
+                {Math.round(parseFloat((this.state.audioFeatures as AudioFeaturesResponse)["danceability"]) * 100)}
+              </div>
             </div>
             <div className={styles.graphicContainer}>
-              {/* <div style={{width: "70px", height: "70px", borderRadius: "35px", backgroundColor: "white"}}></div> */}
+              <CircularProgressbar styles={{path: {stroke: "white"}, trail: {stroke: "rgb(80,80,80)"}}} value={parseFloat((this.state.audioFeatures as AudioFeaturesResponse)["danceability"])} maxValue={1}></CircularProgressbar>
             </div>
           </div>
           {/* Statistic #2 */}
           <div className={styles.statContainer}>
             <div className={styles.statTextContainer}>
-              <div className={styles.text + styles.statLabel} style={{fontSize: "23px", color: "rgb(180,180,180)", fontWeight: "600"}}>{"Energy"}</div>
-              <div className={styles.text + styles.statValue} style={{fontSize: "48px", color: "white", fontWeight: "500"}}>{"55"}</div>
+              <div className={styles.text + styles.statLabel} style={{fontSize: "23px", color: "rgb(200,200,200)", fontWeight: "600"}}>{"Energy"}</div>
+              <div className={styles.text + styles.statValue} style={{fontSize: "48px", color: "white", fontWeight: "500"}}>
+                {Math.round(parseFloat((this.state.audioFeatures as AudioFeaturesResponse)["energy"]) * 100)}
+              </div>
             </div>
             <div className={styles.graphicContainer}>
-              {/* <div style={{width: "70px", height: "70px", borderRadius: "35px", backgroundColor: "white"}}></div> */}
+            <CircularProgressbar styles={{path: {stroke: "white"}, trail: {stroke: "rgb(80,80,80)"}}} value={parseFloat((this.state.audioFeatures as AudioFeaturesResponse)["energy"])} maxValue={1}></CircularProgressbar>
             </div>
           </div>
```

Comment:
Is there a way to make this code more dynamic as well so that we don't have to copy and paste all the HTML and CSS elements each time?